Resources/Prototypes/Entities/Objects/Fun/toys.yml

```diff
@@ -74,6 +74,21 @@
   - type: Sprite
     state: plushie_rouny
 
+- type: entity
+  parent: BasePlushie
+  id: PlushieLamp
+  name: lamp plushie
+  description: A light emitting friend!
+  components:
+  - type: Sprite
+    state: plushie_lamp
+  - type: PointLight
+    netsync: false
+    enabled: false
+    radius: 1.5
+    energy: 2
+  - type: ToggleableLightVisuals
```

Comment:
This is supposed to emit a light on toggle, ye?
I tested it out locally and it doesn't, it just WEH's.
I think the useInHand event is firing only to base plushie and not to ToggleableLight, but don't quote me on that, someone with more coding knowledge can help you with this :D

Reply:
ok thanks